pt/utils/socketio.py

```diff
@@ -0,0 +1,46 @@
+from flask import request
+# pylint: disable=W0622
+from flask_socketio import SocketIO, ConnectionRefusedError
+# pylint: enable=W0622
+from itsdangerous import BadSignature
+
+from utils.oauth import serializer
+from utils.logging import logging
+from endpoints.user.model import User
+
+
+# pylint: disable=C0103
+socketio = SocketIO()
+# pylint: enable=C0103
+
+
+@socketio.on('connect')
+def connected():
+    verified = False
```

Comment:
Maybe is_verified is more accurate to its meaning

Like [this](https://stackoverflow.com/questions/18500718/fql-user-table-is-verified-vs-verified)

Reply:
Fixed. renamed the function to is_verified()